example/spec/parallel/ingestGranule/IngestGranuleWithDuplicateHandlingErrorSpec.js

```diff
@@ -204,6 +208,9 @@ describe('The IngestGranuleCatchDuplicateErrorTest workflow with DuplicateHandli
       { stopOnError: false }
     ).catch(console.error);
 
+    await deleteExecution({ prefix: config.stackName, executionArn: firstIngestGranuleExecutionArn });
+    await deleteExecution({ prefix: config.stackName, executionArn: secondIngestGranuleExecutionArn });
```

Comment:
🐜 I think you can combine these into a Promise.all but it's definitely a nit

Reply:
To save myself from copy 🍝 , this comment is applicable to other tests too

Reply:
Oops, I made some similar comments below 😅         I did suggest promise.allSettled for some of them, but realize we haven't been doing a 'remove everything you can then fail' approach to the afterAlls, so a promise.all would be fine.

Reply:
Yup, in a lot of cases the order of deletes matters but not here. Updated a bunch of these to either use Promise.all or add a note explaining why not:

https://github.com/nasa/cumulus/pull/2252/commits/bf4c4749b365348d77fcaeedef7751611b732428